pre-update-user-profile-samples/pre-profile-update-validation-service-express-choreo/index.js

```diff
@@ -0,0 +1,93 @@
+const express = require('express');
+const nodemailer = require("nodemailer");
+require("dotenv").config();
+
+const app = express();
+const PORT = 3000;
+
+// Middleware to parse JSON request bodies
+app.use(express.json());
+
+// Root endpoint
+app.get('/', (req, res) => {
+    return res.status(200).json({"status": "ok", "message": "Service is running."});
+});
+
+// Mock: valid department list (simulating a directory check)
+const validDepartments = ["Engineering", "HR", "Sales", "Finance"];
+
+// Email transporter config
+const transporter = nodemailer.createTransport({
+    host: "sandbox.smtp.mailtrap.io",
+    port: 2525,
+    auth: {
+        user: process.env.MAILTRAP_USER,
+        pass: process.env.MAILTRAP_PASS
+    }
+});
+
+// Helper to extract claim values
+const getClaimValue = (claims, uri) => {
+    const claim = claims.find(c => c.uri === uri);
+    return claim ? claim.value : null;
+};
+
+app.post("/validate-user-profile-update", async (req, res) => {
+    const payload = req.body;
+
+    if (payload.actionType !== "PRE_UPDATE_PROFILE") {
+        return res.status(200).json({
+            actionStatus: "FAILED",
+            failureReason: "invalid_input",
+            failureDescription: "Invalid actionType provided."
+        });
+    }
+
+    const claims = payload?.event?.request?.claims || [];
+    const userId = payload?.event?.user?.id || "Unknown User";
+
+    const department = getClaimValue(claims, "http://wso2.org/claims/department");
+    const email = getClaimValue(claims, "http://wso2.org/claims/emailaddress");
+    const phone = getClaimValue(claims, "http://wso2.org/claims/mobile");
+
+    // Department validation
+    if (department && !validDepartments.includes(department)) {
+        return res.status(200).json({
+            actionStatus: "FAILED",
+            failureReason: "invalid_department_input",
+            failureDescription: "Provided user department value is invalid."
+        });
+    }
+
+    // Send security alert email if sensitive attributes are being updated
+    const changes = [];
+    if (department) changes.push(`Department: ${department}`);
+    if (email) changes.push(`Email: ${email}`);
+    if (phone) changes.push(`Phone: ${phone}`);
+
+    if (changes.length > 0) {
+        try {
+            await transporter.sendMail({
+                from: '"Security Alert" <security-notifications@wso2.com>',
+                to: "security-team@wso2.com", // Replace with actual security email
```

Comment:
Let's not have this in the sample code. some one running this could trigger mails here. 
Also, can we move this config to some file and in setup instructions let's add steps to update the file